aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -475,9 +492,9 @@ def export(
             **traversal_rules
         )
 
-        output_data = {}
+        report_data: Dict[str, Any] = {'time_write_start': None, 'time_write_stop': None, 'writer_data': None}
```

Comment:
Just out of curiosity - is this the suggested way to build instances of dataclasses, i.e. to build a dict and then pass it to the constructor?

E.g. couldn't you just initialize it below with
`report = ExportReport(time_collect_start=..., time_collect_stop=...)`
and then just add to the instance?

It's not very important here, but in general it would avoid retyping the attributes of the data class as dict keys.

Reply:
well you would have to intialise it with a dummy `time_collect_start` etc, then assume that it will get overriden later on, which I don't think is a good idea

Reply:
no, you could just initialize it once you have both `collect` times

Reply:
ah potatas, potatas, both are valid 😉